src/main/java/com/vladyslavpalamarchuk/trackmycoin/command/processor/AddMonitoringCommandProcessor.java

```diff
@@ -0,0 +1,95 @@
+package com.vladyslavpalamarchuk.trackmycoin.command.processor;
+
+import com.vladyslavpalamarchuk.trackmycoin.adaptors.api.BinanceApiClient;
+import com.vladyslavpalamarchuk.trackmycoin.adaptors.telegram.TelegramBotClient;
+import com.vladyslavpalamarchuk.trackmycoin.command.Command;
+import com.vladyslavpalamarchuk.trackmycoin.service.MonitoringService;
+import java.math.BigDecimal;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.meta.api.objects.Update;
+
+@Component
+@RequiredArgsConstructor
+public class AddMonitoringCommandProcessor implements CommandProcessor {
+
+  private final BinanceApiClient binanceApiClient;
+  private final TelegramBotClient telegramBotClient;
+  private final MonitoringService monitoringService;
+  private final String ADD_COMMAND = "/add";
+  private final int PART_LENGTH = 2;
+
+  @Override
+  public void process(Update update) {
+    Long chatId = update.getMessage().getChatId();
+    String userMessage = update.getMessage().getText();
+    if (userMessage.equalsIgnoreCase(ADD_COMMAND)) {
```

Comment:
you already did this check in commandProcessorRegistry.get by command

Reply:
You can just ask user to input like /add BTC 100000 and then process it inside one command processor